ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -74,7 +74,7 @@ let assert_sr_support_migration ~__context ~vdi_map ~remote_rpc ~session_id =
 			| [ _, plugin ] -> plugin.API.sM_capabilities
 			| _ -> []
 		in
-		if not (List.exists (fun cp -> cp = "Vdi_snapshot") sm_capabilities) then
+		if not (List.exists (fun cp -> cp = "VDI_SNAPSHOT") sm_capabilities) then
```

Comment:
Perhaps you should use `Smint.string_of_capability` then we wouldn't fall into this problem. These magic strings should really only be defined in one place.